walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -245,6 +248,135 @@ public void fetchJob_normalInput_success() throws ExecutionException, Interrupte
         assertEquals(job, actualJob);
     }
 
+    @Test
+    public void fetchAllEligibleJobs_normalInput_success() throws IOException, ExecutionException, InterruptedException {
+        // Arrange.
+        Map<String, Boolean> requirements = ImmutableMap.of(DRIVING_LICENSE_C.getRequirementId(), false,
+                O_LEVEL.getRequirementId(), true, ENGLISH.getRequirementId(), false);
```

Comment:
Maybe some of the test job have less than 3 keys? e.g { english: true, o_level:true }